install/kubernetes/Makefile

```diff
@@ -53,7 +53,10 @@ update-versions:
 	@# Fix up the managed etcd version, as that has its own scheme
 	$(QUIET)sed -i 's/'$(VERSION)'/'$(MANAGED_ETCD_VERSION)'/' $(MANAGED_ETCD_PATH)
 
+lint:
+	$(QUIET)helm lint --with-subcharts --values ./cilium/values.yaml ./cilium
+
 clean:
 	$(RM) $(QUICK_INSTALL) $(EXPERIMENTAL_INSTALL)
 
-.phony: all clean update-versions
```

Comment:
Should we also add `quick-install` / `experimental-install` to the `.PHONY` target?

Reply:
yes, it's my bad. Thanks :100: